web_x2many_delete_all/static/src/js/web_x2many_delete_all.js

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2016 Onestein
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_x2many_delete_all.Mixin', function (require) {
+    "use strict";
+    var core = require('web.core');
+    var _t = core._t;
+    var ListRenderer = require('web.ListRenderer');
+    
+    ListRenderer.include({
+        events: _.extend({}, ListRenderer.prototype.events, {
+            'click thead .o_list_record_delete_all': '_onTrashAllIconClick',
+        }),
+        _renderHeader: function (isGrouped) {
```

Comment:
You can remove the isGrouped parameter here because you don't use it.

Reply:
Done. Thanks !!